googletest/src/assertions.rs

```diff
@@ -1511,8 +1511,7 @@ macro_rules! expect_that {
 
     // general case:
     ($actual:expr, $expected:expr $(,)?) => {{
-        use $crate::GoogleTestSupport as _;
-        $crate::verify_that!($actual, $expected).and_log_failure();
+        $crate::GoogleTestSupport::and_log_failure($crate::verify_that!($actual, $expected));
```

Comment:
Thank you for rebasing! But does it fully solve the problem? There are more instances of `use $crate::GoogleTestSupport as _;` above and below.

Reply:
I pushed two early :laughing: . I'll have more in a moment